### DISCUSSION TITLE
Projects: Undo feature

### DISCUSSION THREAD

**Discussion by steebchen** [community/community#6251] [Projects and Issues]
Maybe I'm the only one being clumsy, but it already happened to me twice that I was typing but accidentally renaming an issue by doing so.

I was hitting CTRL+Z/CMD+Z out of reflex but realised that this doesn't work. I think it'd be great if it would!

**Comment by kmgallahan** [comment]
Same thing just happened to me. Clicked the browser, hit a key for whatever reason, tried to undo, failure and whatever the item was is just gone unless I can somehow remember what it might have been.

**Comment by yck011522** [comment]
Would be nice to be able to undo the action of turning a note to an Issue of a Repo. Or ability to change Issue from Repo to Repo.

**Comment by Flamenco** [comment]
If the project item is an issue, the issue has a timeline record of the change.  Helpful to restore it for now...  Even more helpful than `undo` would be a small `x` to cancel a change before commiting.
